packages/testing/README.md

```diff
@@ -24,12 +24,24 @@ This is part of the default [open-wc](https://open-wc.org/) recommendation
 
 ## Setup
 ```bash
-npx -p yo -p generator-open-wc -c 'yo open-wc:testing-bare'
+npx -p yo -p generator-open-wc -c 'yo open-wc:testing'
 ```
 
+This runs the setup for multiple packages (testing-bare, testing-karma, testing-karma-bs).  
```

Comment:
Instead of describing what these packages do in inline-comments in the code block for the manual section, describe the whole recommendation above in `## Setup`, then add specific install instructions under `### Automatic Setup` and `### Manual Setup`

Reply:
updated it can you pls recheck

Reply:
I would add a new paragraph immediately under `Setup`, and change the heading `Setup` to `Automating Tests`

```
## Automating Tests
Normally, you'll want some way of automatically running all of your tests, for that we recommend karma via @open-wc/testing-karma and browserstack via @open-wc/testing-karma-bs. If you use a different runner or a different browser grid then these steps can be skipped.

### Automatic Install
...
### Manual Install
...
```

Reply:
change it to suggested way